test/e2e/SimulatedInstallationTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RoaveE2ETest\YouAreUsingItWrong;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+
+final class SimulatedInstallationTest extends TestCase
+{
+    /** @var string|null */
+    private $repository;
+
+    protected function tearDown() : void
+    {
+//        if ($this->repository !== null) {
```

Comment:
Kept these cleanups out for now - can restore them later if really needed, but cleaning up directories with PHP is no fun.